.github/workflows/ci.yaml

```diff
@@ -0,0 +1,42 @@
+name: CI
```

Comment:
What is the goal of this? This does not save any file? Just to see that things build?

Reply:
Yes, currently the GitHub Actions only serve as runnable documentation to make sure our build instructions are working.

I saw you are manually maintaining the RP2040 uf2 binary, this could be automated and published to taradov.com [like edbg](https://taradov.com/bin/edbg/) if you are interested.